packages/block-library/src/cover/edit.native.js

```diff
@@ -106,9 +107,32 @@ const Cover = ( {
 		customOverlayColor,
 		minHeightUnit = 'px',
 	} = attributes;
+	const [ isScreenReaderEnabled, setIsScreenReaderEnabled ] = useState(
+		false
+	);
 
 	const CONTAINER_HEIGHT = minHeight || COVER_DEFAULT_HEIGHT;
 
+	useEffect( () => {
+		// sync with local media store
+		mediaUploadSync();
```

Comment:
Please remove it, or the other `useEffect` which invoke `mediaUploadSync` here :)
https://github.com/WordPress/gutenberg/pull/26727/files#diff-efe4dc27db844b16612461b3acc8ac6d6ee9c44a490baf9bb58c03e6f88b77d7R181